packages/cashier/src/pages/withdrawal/withdrawal-crypto-form/withdrawal-crypto-form.scss

```diff
@@ -28,17 +40,24 @@
     }
     &__percentage {
         &-container {
+            width: 100%;
             margin-top: 2.4rem;
         }
         &-selector {
             margin-bottom: 1.6rem;
-            width: fit-content;
+            width: 100%;
             @include mobile {
                 display: flex;
                 flex-direction: column;
                 align-items: center;
-                width: 100%;
             }
         }
     }
 }
+
+.cashier__wrapper--is-mobile .cashier__wrapper {
+    @include mobile {
+        width: 100%;
+        margin-inline: 0;
+    }
+}
```

Comment:
We have to override this outside of the root CSS class, it doesn't work otherwise.

Reply:
question: would it fit better into .cashier.scss file?

Reply:
You're right @wojciech-deriv, I've removed this code entirely though, in favour of creating a new local class.